src/monodroid/jni/dylib-mono.h

```diff
@@ -295,10 +269,10 @@ struct DylibMono {
 	monodroid_mono_method_full_name_fptr                    mono_method_full_name;
 	monodroid_mono_object_get_class_fptr                    mono_object_get_class;
 	monodroid_mono_object_unbox_fptr                        mono_object_unbox;
-	monodroid_mono_profiler_install_fptr                    mono_profiler_install;
-	monodroid_mono_profiler_install_jit_end_fptr            mono_profiler_install_jit_end;
-	monodroid_mono_profiler_install_thread_fptr             mono_profiler_install_thread;
-	monodroid_mono_profiler_set_events_fptr                 mono_profiler_set_events;
+	monodroid_mono_profiler_install_fptr                     mono_profiler_install;
+	monodroid_mono_profiler_set_jit_done_callback_fptr       mono_profiler_set_jit_done_callback;
```

Comment:
I expect to resurrect MonoUE's Android support at some point but it currently has no users, so breakage doesn't matter.